### DISCUSSION TITLE
Using Omniauth with a separate Javascript (react) frontend

### DISCUSSION THREAD

**Discussion by jgordo04** [omniauth/omniauth#1114] [Q&A]
Hi. I have been using omniauth for google oauth2 successfully for quite a while. We are using it with a rails slim frontend page. We are now trying to convert this to work with our react frontend login, but having a lot of trouble with the authenticity token and now just a 403 with `https://accounts.google.com/v3/signin/identifier` even before logging in with google. I'm kind of at a loss for what to troubleshoot at this point. It seems omniauth may put restrictions on outside entities using its routes? If anyone has any experience with this or any ideas, I would love to discuss.

**Comment by preciousuwhubetine** [comment]
Were you able to get this working?

**Reply by jgordo04** [reply]
No. We actually went with using the google-auth gem when working with our react frontend instead.
